sse2neon.h

```diff
@@ -2618,6 +2618,22 @@ FORCE_INLINE __m128i _mm_min_epi32(__m128i a, __m128i b)
         vminq_s32(vreinterpretq_s32_m128i(a), vreinterpretq_s32_m128i(b)));
 }
 
+// Compare packed unsigned 32-bit integers in a and b, and store packed maximum
+// values in dst.
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_max_epu32
+FORCE_INLINE __m128i _mm_max_epu32(__m128i a, __m128i b)
+{
+    return (__m128i) vmaxq_u32((uint32x4_t) a, (uint32x4_t) b);
```

Comment:
Change `(__m128i)` to `vreinterpretq_m128i_u32`.
Change `(uint32x4_t)` to `vreinterpretq_u32_m128i`.